shopfloor/actions/message.py

```diff
@@ -421,6 +421,28 @@ def location_content_unable_to_transfer(self, location_dest):
             ).format(location_dest.name),
         }
 
+    def product_in_multiple_sublocation(self, product):
+        return {
+            "message_type": "warning",
+            "body": _(
+                "Product {} found in multiple locations. Scan your location first."
+            ).format(product.name),
+        }
+
+    def lot_in_multiple_sublocation(self, lot):
+        return {
+            "message_type": "warning",
+            "body": _(
+                "Lot {} for product {} found in multiple locations. Scan your location first."
+            ).format(lot.name, lot.product_id.name),
```

Comment:
Not blocking, but term places could be swapped in some languages, better to use keys to place them:
```suggestion
            "body": _(
                "Lot {lot} for product {product} found in multiple locations. Scan your location first."
            ).format(lot=lot.name, product=lot.product_id.name),
```

Reply:
You have a point, I added the change.